graylog2-server/src/main/java/org/graylog2/inputs/GraylogSchemaFields.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * This file is part of Graylog.
+ *
+ * Graylog is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Graylog is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Graylog.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.graylog2.inputs;
```

Comment:
What do you think about moving this into the `org.graylog.schema` package? (doesn't exist yet) I think we will have more related code in there in the future and so it isn't buried in the inputs package. :smiley:

Reply:
Great idea.  That will promote visibility and re-use.  I'll make the change.